docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/akamai-siem-api-source.md

```diff
@@ -50,7 +50,7 @@ To configure an Akamai SIEM API Source:
 1. In Sumo Logic, select **Manage Data** > **Collection** > **Collection**. 
 1. On the Collection page, click **Add Source** next to a HostedCollector.
 1. Select **Akamai SIEM API**. <br/>   ![Akamai source icon.png](/img/send-data/Akamai-source-icon.png)
-1. Enter a **Name** to display for the Source in the Sumo web application. The description is optional.<br/>   ![Akamai SIEM API Source](/img/send-data/Akamai-SIEM-API-Source.png)
+1. Enter a **Name** to display for the Source in the Sumo web application. The description is optional.<br/>   ![Akamai SIEM API Source](/img/send-data/Akamai-SIEM-API-Source.png =300x)
```

Comment:
This won't work. Renders like this.

<img width="361" alt="Screenshot 2023-04-30 at 10 14 45 AM" src="https://user-images.githubusercontent.com/56411016/235366906-6fd28edb-be09-42eb-b9d5-c48a1d842b8c.png">

If you want to change the image size, you need to convert it to HTML syntax (not markdown).
`<img src={useBaseUrl('img/send-data/Akamai-SIEM-API-Source.png')} alt="Thumbnail icon" width="300"/>`

Reply:
@kimsauce Got it. Thank you.